piezo_web_app/PiezoWebApp/tests/integration_tests/tidy_jobs_test.py

```diff
@@ -113,9 +113,9 @@ def test_post_successfully_writes_logs_and_deletes_completed_and_failed_jobs(sel
         assert response_code == 200
         self.mock_logger.debug.assert_any_call('Not processing job "job1", current status is "RUNNING"')
         self.mock_logger.debug.assert_any_call('Logs written to "outputs/job2/log.txt"')
-        self.mock_logger.debug.assert_any_call('Trying to delete job "job2" resulted in status: 200')
+        self.mock_logger.debug.assert_any_call('Trying to delete job "job2" resulted in status: 200\n')
```

Comment:
The final `\n` isn't ideal... can we avoid this somehow? Maybe gather together the messages for logging/returning as a list:
* logging: loop through them, logging each one separately
* returning: join them by `'\n'.join(messages)`